Tests/fixtures/podcasting.yml

```diff
@@ -9069,3 +9069,335 @@
+    platform: ""
+  client:
+    type: mobile app
+    name: Youtube Music
+    version: "62354340"
+  device:
+    type: smartphone
+    brand: Google
+    model: 'Pixel 7 Pro'
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'YouTubeMusic/20231020.1.1 CFNetwork/1474 Darwin/23.0.0'
+  os:
+    name: iOS
+    version: "17.0"
+    platform: ""
+  client:
+    type: mobile app
+    name: Youtube Music
+    version: "20231020.1.1"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'YouTubeMusic/6.29.2 CFNetwork/1485 Darwin/23.1.0'
+  os:
+    name: iOS
+    version: "17.1"
+    platform: ""
+  client:
+    type: mobile app
+    name: Youtube Music
+    version: "6.29.2"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Amazon;Echo_Dot;27d4dfe427b34d57995b463e5d63198d;;tpapi;3.199.422'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Alexa Media Player
+    version: ""
+  device:
+    type: smart speaker
+    brand: Amazon
+    model: 'Echo Dot'
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Amazon;Echo_Dot_with_clock;27d4dfe427b34d57995b463e5d63198d;;tpapi;3.199.422'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Alexa Media Player
+    version: ""
+  device:
+    type: smart speaker
+    brand: Amazon
+    model: 'Echo Dot'
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Amazon;Echo_Show_5;27d4dfe427b34d57995b463e5d63198d;;tpapi;3.199.422'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Alexa Media Player
+    version: ""
+  device:
+    type: tablet
+    brand: Amazon
+    model: 'Echo Show 5'
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Amazon;Echo;27d4dfe427b34d57995b463e5d63198d;;tpapi;3.199.422'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Alexa Media Player
+    version: ""
+  device:
+    type: smart speaker
+    brand: Amazon
+    model: Echo
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'DS%20Podcast/2 CFNetwork/1410.0.3 Darwin/22.6.0'
+  os:
+    name: iOS
+    version: "16.6"
+    platform: ""
+  client:
+    type: mobile app
+    name: De Standaard
+    version: "2"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'DS podcast/1.0.4 (be.standaard.audio; build:5; Android 11; Sdk:30; Manufacturer:samsung; Model: SM-A505FN) OkHttp/4.9.0'
+  os:
+    name: Android
+    version: "11"
+    platform: ""
+  client:
+    type: mobile app
+    name: De Standaard
+    version: "1.0.4"
+  device:
+    type: smartphone
+    brand: 'Samsung'
+    model: 'Galaxy A50'
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'SM-T970 (compatible; Tablet2.0) HandelsbladProduction, com.twipemobile.nrc 5.1.4 (511) / Android 33'
+  os:
+    name: Android
+    version: "33"
```

Comment:
The version here is wrong and it refers to `Android API 33` -> `Android 13`. I wanted to fix this a while ago. If you want to fix it, is great! Another user agent to be added as test:

`MXPlayer/1.73.3/2001002183; Android/33`

Reply:
I put in a fix for this, I hope duplicating the android block is acceptable, I don't think trying to combine those top level regexes seems like a very good idea.